instrumentation/spring/spring-boot-actuator-autoconfigure-2.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/spring/actuator/SpringBootActuatorInstrumentationModule.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.spring.actuator;
+
+import com.google.auto.service.AutoService;
+import io.opentelemetry.javaagent.extension.instrumentation.HelperResourceBuilder;
+import io.opentelemetry.javaagent.extension.instrumentation.InstrumentationModule;
+import io.opentelemetry.javaagent.extension.instrumentation.TypeInstrumentation;
+import java.util.Collections;
+import java.util.List;
+
+@AutoService(InstrumentationModule.class)
+public class SpringBootActuatorInstrumentationModule extends InstrumentationModule {
+
+  public SpringBootActuatorInstrumentationModule() {
+    super("spring-boot-actuator-autoconfigure", "spring-boot-actuator-autoconfigure-2.0");
+  }
+
+  @Override
+  public void registerHelperResources(HelperResourceBuilder helperResourceBuilder) {
+    // autoconfigure classes are loaded as resources using ClassPathResource
+    // this line will make OpenTelemetryMeterRegistryAutoConfiguration available to all
+    // classloaders, so that the bean classloader (different than the instrumented classloader) can
+    // load it
+    helperResourceBuilder.registerForAllClassLoaders(
+        "io/opentelemetry/javaagent/instrumentation/spring/actuator/OpenTelemetryMeterRegistryAutoConfiguration.class");
```

Comment:
I used a bit different way of making that class visible to spring class loaders; maybe it'll be useful in other spring instrumentations too.

(Also, an interesting bit: no need to explicitly pass `.classdata` here, the `AgentClassLoader` will add the suffix automatically when it's asked to get a `.class` resource)

Reply:
💯